src/Alert/Alert.js

```diff
@@ -7,24 +7,22 @@ import { getComponentVariant, createComponent } from '../utils';
 const StyledAlert = createComponent({
   name: 'Alert',
   style: ({ variant, theme }) => {
-    const { backgroundColor, fontColor } = getComponentVariant(theme, 'Alert', variant);
+    const variantStyles = getComponentVariant(theme, 'Alert', variant);
 
     return css`
       padding: 1rem;
       margin-bottom: 1rem;
       border: 0;
       font-size: 14px;
       font-family: ${theme.typography.fontFamily || 'inherit'};
-      border-radius: ${theme.elementRadius}px;
-      border-left: 4px solid ${fontColor};
```

Comment:
Did we decide to remove the border-left on alerts? Sorry if I missed this thread.